mantidimaging/gui/windows/spectrum_viewer/spectrum_widget.py

```diff
@@ -134,6 +134,8 @@ def add_roi(self, roi: SensibleROI, name: str) -> None:
         self.max_roi_size = roi_object.size()
         self.roi_dict[name].sigRegionChanged.connect(self.roi_changed.emit)
         self.image.vb.addItem(self.roi_dict[name])
+        # On hover, the ROI border turns red and line thickens
+        self.roi_dict[name].hoverPen = mkPen((255, 51, 51), width=3)
```

Comment:
Would it work for the colour to stay the same, so the line just gets thicker?

Reply:
Yes, that's fine by me! I tried that in my first attempt, but preferred the change in colour as it made the selected/hovered ROI more noticeable to me.  I remain pretty neutral on it myself though so will make the change now 🙂

Reply:
I've played a bit with both, and still leaning to towards keeping colour the same.